src/global.spec.ts

```diff
@@ -0,0 +1,29 @@
+import client from "./shared/api/cbioportalClientInstance";
+import Sinon from "sinon";
+import CBioPortalAPI from "./shared/api/generated/CBioPortalAPI";
+
+let clientStubs:any[];
+before(()=>{
+    clientStubs = stubClient();
+});
+
+after(()=>{
+    for (const stub of clientStubs) {
+        stub.restore();
+    }
+});
+
+function stubClient():any[] {
+    // returns stubs
+    const stubs:any[] = [];
+    const methodNames:(keyof CBioPortalAPI)[] = (Object.keys(client) as (keyof CBioPortalAPI)[]).filter(m=>(typeof (client as any)[m]) === "function");
+    for (const methodName of methodNames) {
```

Comment:
there are a couple methods on client that aren't fetchers, so you might want to only stub /^get|fetch/

Reply:
oh ok

Reply:
example?